xds/src/main/java/io/grpc/xds/sds/XdsChannelBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.sds;
+
+import io.grpc.ForwardingChannelBuilder;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.netty.NettyChannelBuilder;
+import io.grpc.xds.sds.internal.SdsProtocolNegotiators;
+import java.net.SocketAddress;
+import javax.annotation.CheckReturnValue;
+
+/**
+ * A wrapper around NettyChannelBuilder that sets up the appropriate ProtocolNegotiator
+ * and ChannelHandlers to use SDS for TLS.
```

Comment:
this doc just describes the implementation detail, should describe the function of class like AltsChannelBuilder. also, applies to the server builder. also the doc should mention that this is not ready to use, etc.